ts/RailroadMap.ts

```diff
@@ -1098,7 +1105,7 @@ export class RailroadMap {
                 if (this.gizmoDebugText) this.gizmoDebugText.remove();
                 this.gizmoDebugLine = gizmoG.line().addClass('ruler');
                 this.gizmoDebugText = gizmoG
-                    .text(`[${x}, ${y}]`)
+                    .text(`[${x}, ${y}]`) // embix: how to enable/test this?
```

Comment:
This one always displays the text in the same orientation relative to the industry because the text is part of the industry's group, so don't worry about updating this one in this PR. That's something that I should fix, but it's not a priority.

This is part of the industry measurement tool. You can enable it from the layers dropdown in the map, and use it by clicking and dragging from the arrows that appear over each industry's origin point. It's used for measuring coordinates in the industry's local coordinate space.

The orientation doesn't matter so much here because the coordinates are logged to the console on mouse up.